packages/react/src/themes/teams/components/Input/variants/index.ts

```diff
@@ -0,0 +1,7 @@
+import Minimalistic from './Minimalistic'
```

Comment:
these temporary changes are introduced for the sake of example to reviewer - won't be merged